Gems/AWSCore/Code/Source/ResourceMapping/AWSResourceMappingManager.cpp

```diff
@@ -244,14 +245,16 @@ namespace AWSCore
 
     bool AWSResourceMappingManager::ValidateJsonDocumentAgainstSchema(const rapidjson::Document& jsonDocument)
```

Comment:
IMHO: Json schema validation should be central util and not part of AWSResourceMapping manager. It can have a ValidateMappingFile() but that should just set up paths (ie thing to validate, validation scheme)

Reply:
I think your comment doesn't make much sense to me, json schema validation should be part of AWSResourceMapping manager, as it has the responsibility to take care of all work related to resource mapping stuff. I don't see a value that other components in AWSCore requires to access resource mapping json schema.
I can create have a helper function ValidateMappingFile, but as I said it should be a private function. I don't see a value that developer/other components need to call it separately and specifically outside.